### DISCUSSION TITLE
Simple Notifications (without MQTT)

### DISCUSSION THREAD

**Discussion by mjm987** [blakeblackshear/frigate#10331] [Ideas]
I'm using Frigate via Docker standalone without HA.
For a external event notification (eg. switching on lights in case of a event) I'm looking for a simple solution without MQTT.
Is there a easy way to start eg. a shell script or a simple HTTP POST/PUT in case of a event?

**Comment by NickM-27** [comment]
MQTT or websocket is the main way to get on demand data about what frigate is detecting. You could do polling of http but that would either load down the system or have delay.

**Reply by mjm987** [reply]
Thanks for suggestion. Websocket might be a suitable solution but I could not find a documentation.
I guess I have to study the javascript part of the source code or is there any other documentation available?

**Reply by NickM-27** [reply]
the websocket is not documented in terms of connecting to it. The data however is identical to what is sent over mqtt

**Reply by mjm987** [reply]
Ok, I will try. 
Thx

**Reply by mjm987** [reply]
Because I have no knowledge about websocket and thus didn't get working quickly, I tried it by using a simple Python MQTT server protocol implementation found here: [qmqtt](https://github.com/qingyunha/qmqtt.git)
... and triggered on topic `frigate/events`
